app/move/controllers/update/base.js

```diff
@@ -0,0 +1,114 @@
+const { get, keys } = require('lodash')
+
+const personService = require('../../../../common/services/person')
+const CreateBaseController = require('../create/base')
+
+class UpdateBaseController extends CreateBaseController {
+  middlewareLocals() {
+    super.middlewareLocals()
+    this.use(this.setStepUrls)
+    this.use(this.setInitialStep)
+  }
+
+  _setModels(req) {
+    const res = req.res
```

Comment:
This feels a little strange? Is it to avoid passing `res` around to?

I think I had something else in mind when I said to make use of the middleware functions that the form wizard provides. Rather than creating extra getters/setters I though it could work to just set the values on `req` instead of having to create get methods on req and create a set of models on req.

Reply:
OK - will revisit in editabilty part deux aka STC/SCH users